docs/sphinx/source/contributing.rst

```diff
@@ -309,22 +309,22 @@ To run the tests locally, install the ``test`` dependencies specified in the
 file. See :ref:`installation` instructions for more information.
 
 pvlib's unit tests can easily be run by executing ``pytest`` on the
-pvlib directory:
+pvlib directory::
 
-``pytest pvlib``
+    $ pytest pvlib
```

Comment:
Thanks for fixing the formatting here. 

At the risk of bike shedding... Do we want to use `$` for terminal prompts? I don't know if that's more or less clear to the relevant readers. We don't currently use them in other sections of the documentation, e.g. https://pvlib-python.readthedocs.io/en/stable/installation.html#install-standard-release Up to you, just asking.

Reply:
I'm happy to remove it, but IMO it's a good question. I think one good reason to use a prompt, is to distinguish between the terminal (CMD: `>`, BaSH: `$`, etc) and a Python interpreter: `>>>` -- which might seem obvious to one of us, but I have found it is a frequent stumbling block for new users. We say, "type `pytest pvlib`," but not where, so they type this in Python, instead of a terminal.

I think we can be explicit every time, and say, "enter this into a terminal ," or another option is to say once, "the `$` symbol is used to indicate that the commands are entered in a terminal, and the `>>>` symbol indicates this is used in a Python interpreter."

**Perhaps we should make this a separate issue, that can be discussed by the community, and then addressed globally in all of the docs, with a preamble like I suggested above?**

Reply:
Sounds good to me. Consistency with python/numpy/pandas documentation is probably the best approach, but I don't know how consistent they are within their own documentation or across those organizations.

Reply:
I posted a [google group question](https://groups.google.com/forum/#!topic/pvlib-python/BT3Mjerp92g), because I don't want to make an issue if this is a non-issue

PS on [a different topic](https://groups.google.com/d/msg/pvlib-python/aAcnD97KsEk/QALrIVsnAQAJ), I made a [test forum](https://pvlib-python.alwaysdata.net/discuss/) to try out